client.go

```diff
@@ -40,7 +45,10 @@ func NewClientImpl(deployFile string, targetName string) (*ClientImpl, error) {
 		return nil, fmt.Errorf("Invalid target: '%s'", targetName)
 	}
 
-	localPort := setupChannel(target.Base, target.SshPort, target.Ssh)
+	localPort := target.Base
+	if !isLocalTest {
+		localPort = setupChannel(target.Base, target.SshPort, target.Ssh)
```

Comment:
It would be good to abstract the parts that differ if isLocalTest is true and have setupChannel() return something that handles that difference. E.g.

channel := setupChannel(target, isLocalTest)
channel.ExecCmd(...)
channel.CopyTo("/some/local/address", "/some/remote/address")
serverAddr := fmt.Sprintf("localhost:%d", channel.LocalPort)

This would also solve the problem currently where setupChannel() starts an ssh process and then leaves it hanging by returning something we can .Close() (though feel free to add a TODO for that)